addons/web_editor/static/src/components/media_dialog/file_selector.xml

```diff
@@ -1,40 +1,39 @@
 <?xml version="1.0" encoding="utf-8"?>
 <templates id="template" xml:space="preserve">
 <t t-name="web_editor.FileSelectorControlPanel" owl="1">
-    <div class="row g-2 align-items-center py-4">
-        <SearchMedia searchPlaceholder="props.searchPlaceholder" needle="props.needle" search="props.search"/>
-
-        <div class="col-auto d-flex gap-2 align-items-center">
-            <div t-if="props.showOptimizedOption" class="flex-shrink-0 form-check form-switch align-items-center" t-on-change="props.changeShowOptimized">
-                <input class="o_we_show_optimized form-check-input" type="checkbox" t-att-checked="props.showOptimized" id="o_we_show_optimized_switch"/>
-                <label class="form-check-label" for="o_we_show_optimized_switch">
-                    Show optimized images
-                </label>
+    <div>
+        <div class="d-flex flex-wrap gap-2 mt-4 mb-2 align-items-center">
```

Comment:
Why is there an extra div in the hierarchy?

Reply:
> After this PR:
The problem was already there on master, but now it is fixed 👍

> After this PR, go to the image tab, click "Add URL" to show the input, switch to Document tab, a mini URL stays next to "Add document":
Fixed as well 

> Not due to your branch but would be cool to restore if possible:
Unfortunately I was not able to manage this issue, I will need help on this if we want to fix it as part of the scope of this task

> As a forum user:
Fixed too

> Why is there an extra div in the hierarchy?
Div was removed

cc @stefanorigano

Reply:
> The problem was already there on master, but now it is fixed +1

I assure you I don't take time to forge fake screenshot :smile: 

> Unfortunately I was not able to manage this issue, I will need help on this if we want to fix it as part of the
scope of this task

Ok, we'll make the fix on our side

Thanks, @stefanorigano I'll trust you with the merge of this one :+1:

Reply:
> I assure you I don't take time to forge fake screenshot 

🤣 I think that @anso-odoo meant that the conditions exposing the issue in his branch (missing flex-wrap) were already there.

Thanks @qsm-odoo 👍 


Master

https://user-images.githubusercontent.com/8558351/188917555-c19b5a09-4666-41bb-b7c4-45d27de01f49.mov

After the PR

https://user-images.githubusercontent.com/8558351/188917877-df64028f-2b30-4645-a64f-7e1065125e04.mov

Reply:
> I assure you I don't take time to forge fake screenshot

I'm sure you don't, i just wanted to clarify that it was not my PR that created this unwanted behaviour. I should have expressed myself better i'm sorry @qsm-odoo 😅

Reply:
My comment was just a joke, nothing to be sorry for :)